src/extractor/extractor_callbacks.cpp

```diff
@@ -78,10 +78,23 @@ void ExtractorCallbacks::ProcessNode(const osmium::Node &input_node,
     {
         external_memory.barrier_nodes.push_back(id);
     }
-    if (result_node.traffic_lights != TrafficLightClass::NONE)
+    if (result_node.traffic_lights != TrafficFlowControlNodeDirection::NONE)
     {
         external_memory.external_traffic_signals.push_back({id, result_node.traffic_lights});
     }
+    // TODO: we ignore `ALL` for both stop signs and give way signs, because we cannot understand
```

Comment:
I have an idea how to implement it(at least for vast majority of cases), but I'd propose to do in separate PR.

Reply:
Why can't we understand all-way stop/give-ways?

Can we treat them the same as traffic lights without direction?